provisioning/roles/kuma/files/vagrant/settings_local.py

```diff
@@ -108,3 +108,7 @@
     )
 
 SOCIALACCOUNT_PROVIDERS['persona']['AUDIENCE'] = 'https://developer-local.allizom.org'
+
+# Don't cache non-versioned static files in DEBUG mode
+if DEBUG:
+    WHITENOISE_MAX_AGE = 0
```

Comment:
We [override the default](https://github.com/mozilla/kuma/pull/3642/files#diff-bdf3ecebd8379ca98cc89e545fc90899R557) in *settings.py*. These lines override that value when `DEBUG = True`.